config/main.tf

```diff
@@ -0,0 +1,39 @@
+terraform {
+  backend "s3" {
+    key = "terraform/build/callisto-auth-keycloak"
+  }
+}
+
+provider "keycloak" {
+  # Variables in here must be defined locally before running the plan
+  client_id     = var.tf_client_id
+  client_secret = var.tf_client_secret
+  realm         = var.tf_realm
+  url           = var.tf_url
+}
+
+data "keycloak_realm" "callisto_notprod_auth" {
+  realm = "callisto-dev"
```

Comment:
This would need to be a variable as we expect to run the same configuration against each realm as we promote the configuration through environments

Reply:
Changed